tests/python/rest_api/test_projects.py

```diff
@@ -458,6 +458,23 @@ def test_admin_can_get_project_backup_and_create_project_by_backup(self, admin_u
             )
             assert response.status == HTTPStatus.ACCEPTED
 
+    def test_can_import_export_dataset_with_datumaro_format(self):
+        # https://github.com/opencv/cvat/issues/4410
+        username = "admin1"
+        format_name = "Datumaro 1.0"
+        project_id = 4
+
+        response = self._test_export_project(username, project_id, format_name)
+
+        tmp_file = io.BytesIO(response.data)
+        tmp_file.name = "dataset.zip"
+
+        import_data = {
+            "dataset_file": tmp_file,
+        }
+
+        self._test_import_project(username, project_id, format_name, import_data)
+
```

Comment:
I don't think it makes sense to test this twice. It should already be tested in `cvat/apps/dataset_manager/tests/test_rest_api_formats.py`, which is also modified in this PR.

Reply:
I think this test is needed. It refers to a specific issue. The test as here is the most understandable and transparent.

Reply:
As I see, this test is also a complete duplicate of the test above. Maybe we can at least merge their codes and add a parameter?